react/components/ProductPriceTotal.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react"
+import { Spinner } from 'vtex.styleguide'
+import useQueryGetProductPrice from '../hooks/actions/useQueryGetProductPrice'
+
+import styles from './productPriceTotal.css'
+
+const ProductPriceTotal = ({ skuReference = 0, productQuantity = 0, currency = '$' }) => {
+
+  const { refetch, data, error: productError, loading } = useQueryGetProductPrice('reference', skuReference)
```

Comment:
we should improve this hook to return price, error and loading state only.

Reply:
Done